DonjinKrawler-Client/src/main/java/donjinkrawler/items/DamagePotion.java

```diff
@@ -19,8 +27,21 @@ public String getDescription() {
         return "Adds some damage buff.";
     }
 
+    @Override
+    public void accept(ItemVisitor v) {
```

Comment:
From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Use intention revealing names.`